README.md

```diff
@@ -659,6 +659,9 @@ this.addEventListener('.someClass', 'scroll', fn);
 // Attach to a DOM node
 this.addEventListener(document.body, 'click', fn);
 
+// Attach to a component's root element
+this.addEventListener(this.element, 'scroll', fn);
```

Comment:
I'm not opposed to adding this generally, though it makes me wonder if we can do things a bit better for this specific case. 

I wonder if we should have an overload that assumes if no element is passed then we mean `this.element`.

```
this.addEventListener('scroll', fn);
```

Would this be useful?

Reply:
That makes a lot of sense to me.

Reply:
I'm going to bundle these changes up with part of a larger refactor I'm working on. I'll comment on this PR once I've made progress.